internal/core/input.rs

```diff
@@ -679,11 +681,10 @@ pub(crate) fn handle_mouse_grab(
         _ => {
             mouse_input_state.grabbed = false;
             // Return a move event so that the new position can be registered properly
-            Some(
-                mouse_event
-                    .position()
-                    .map_or(MouseEvent::Exit, |position| MouseEvent::Moved { position }),
-            )
+            Some(mouse_event.position().map_or(MouseEvent::Exit, |position| MouseEvent::Moved {
+                position,
+                is_touch: false,
```

Comment:
We should try to preserve the `is_touch` of the original event here. Although it probably doesn't matter too much.